### ISSUE TITLE
md-autocomplete directive does not support tabindex

### ISSUE THREAD

**Issue by mcnocopo** [angular/material#2113]
md-autocomplete directive does not pass the tabindex to the "compiled" inner input element which seem be set to 0.

The following does not work
````
<md-autocomplete
    tabindex="3"
    ng-disabled="vm.user.isDisabled"
    ....
```

Any suggestions?

**Comment by marcysutton** [comment]
Sounds like a missing feature. Existing `tabindex` values should be passed to the input. 

Just a warning, managing the tab order of an entire app can be challenging. That said, our components should work with native attributes as much as possible. That's a valid use case, challenging or not.

**Comment by sp90** [comment]
:+1:  on this one

**Comment by robertmesserle** [comment]
As far as I can tell, `tabindex` works properly on autocomplete.  @marcysutton can you help confirm this?  As is, adding `tabindex` to `md-autocomplete` seems to work properly without transferring the attribute to the input tag.  (Focusing `md-autocomplete` forwards focus to the input.)  Is there any reason to manually transfer this attribute?

**Comment by robertmesserle** [comment]
Closing for now, will reopen if this step is necessary.

**Comment by marcysutton** [comment]
The request was for custom `tabindex` values, which need to be manually moved to the input. Right now, if you add `tabindex="3"` to `md-autocomplete` it remains there, causing both the wrapper and the input to be focusable. Only the input should be focusable.

**Comment by robertmesserle** [comment]
@marcysutton Since `md-autocomplete` is tabbable, and forwards its tabs to the input, would adding `tabindex='-1'` resolve this?

**Comment by marcysutton** [comment]
That looks good to me, assuming focus is forwarded to the input since it is the element with a role.

**Comment by tucq88** [comment]
Is this fixed ? I still got this problem with 1.0.1, both `md-autocomplete` and the input inside have same tabindex attribute
